routes/api/v1/playlists.js

```diff
@@ -151,6 +150,37 @@ router.delete('/:playlistID/favorites/:favoriteID', async (request, response) =>
   }
 });
 
+router.get('/:playlistID/favorites', async (request, response) => {
+  let playlists = await database('playlists')
+    .where('id', request.params.playlistID);
+
+  if (playlists.length) {
+    let favorites = await getFavorites(playlists);
+
+    let playlistsArray =  playlists.map((playlist, index) => {
```

Comment:
Since there is only one playlist returned, we don't necessarily have to map to create the response object. This could be a future refactor.